docs/connecting-to-aws.md

```diff
@@ -1,30 +1,43 @@
 # Connecting to AWS from the AWS Toolkit for Visual Studio Code
```

Comment:
Let's make sure most of this information is in the AWS docs since this won't be surfaced to the user.

Reply:
I plan to engage the doc team after this PR is completed